docs/setting-up/server/docker.md

```diff
@@ -246,7 +246,7 @@ Use the following Docker container environment variables (with `-e var=value`) t
 
 | Variable                   | Description
 | -------------------------- | -----------------------------------------------------------------------
-| `DISABLE_UPDATES`          | Disable automatic updates.
+| `DISABLE_UPDATES`          | Disable automatic checks for updated PMM versions. *Updates are not automatically applied*
```

Comment:
`automatic` is not ideal for me as that could lead to the assumption that checks for updates are still possible -even if not automatic - , which is not the case.

I'd phrase that like `Disable the ability to check for new PMM versions and apply upgrades using the UI`